packages/openactive-integration-tests/test/helpers/request-state.js

```diff
@@ -23,15 +27,16 @@ function isResponse(response) {
 }
 
 class RequestState {
-  constructor (logger) {
+  /**
+   * 
+   * @param {InstanceType<import('./logger')['Logger']>} logger 
+   * @param {string | null} uuid
```

Comment:
It is now possible to instantiate a RequestState with a `uuid`. See my comment in `amend-c1-test.js` to see why